### DISCUSSION TITLE
Responsive UI for Application page

### DISCUSSION THREAD

**Discussion by ghost** [elementary/appcenter#2038] [Ideas]
Current application page in app store looks fine when maximised. However a lot of information is missing when the window is scaled down, and you need to scroll more.

I propose:
- Adapting the app bar to occupy less space.
- Move the "non-curated" label somewhere else or make it occupy less space.
- Make screenshots occupy less space when window is scaled down
- Make tags expandable except the current one as it's not very relevant info anyway.
- Move website, repository, issues, etc. to the top

BEFORE:

![Screenshot from 2023-05-03 11 20 04](https://user-images.githubusercontent.com/46191980/235878993-9166236c-72cc-4493-bece-e2e40d7d3810.png)

AFTER:

![Mockup](https://user-images.githubusercontent.com/46191980/235882251-b4e3278e-221d-421b-82d5-f2917d7f451e.png)

Excuse the terrible GIMP skills..

**Comment by danirabbit** [comment]
Can you be more specific about what you mean by "browsing is hard". What specifically makes it hard? Can you provide supporting screenshots of the problem? Your proposed mockup seems to show a maximized window, but the proposal you're describing is about small windows, so could you provide a mockup that supports your proposal?

**Comment by ghost** [comment]
I made the issue more clear and added a dirty mockup @danirabbit 

Without changing much, just making the appbar bigger and moving the "non curated" label (Only when window is scaled down or there is not enough resolution) already makes the window more browsable.

**Comment by danirabbit** [comment]
I've proposed #2024 to tighten up vertical spacing a bit

I don't think links are important enough to be at the very top of the page. They should probably stay at the bottom of the page since they're probably the least important information for deciding if you want to install an app.

I do think content and permissions warnings are incredibly important and should remain above the fold since this information is requisite for informed consent and could be make or break in deciding if you want to install an app or game.

I also don't think we should be reducing the size of screenshots so much, they're probably the most important information that someone wants to see before deciding if they should install an app. Screenshots can also have captions and in mainstream app stores this is some of the primary copy that developers rely on to advertise the features of their app, so there needs to be room to display those captions.

**Reply by ghost** [reply]
I don't think screenshots should be as little as on the mockup, however it varies from display to display. For a 4K display, maybe they don't occupy that much space. However for a laptop I think they should adapt to leave space for other elements. Because as I see them they're blocking the whole UI right now.

**Comment by danirabbit** [comment]
I'm going to convert this to a discussion since there isn't like a single clear actionable report here
